releases/v20.2.0.md

```diff
@@ -0,0 +1,139 @@
+Core | **User-Defined Schemas** | You can now [create user-defined logical schemas](../v20.2/create-schema.html), as well [alter user-defined schemas](../v20.2/alter-schema.html), [drop user-defined schemas](../v20.2/drop-schema.html), and [convert databases to user-defined schemas](../v20.2/convert-to-schema.html).
+Core | **Partial Indexes** | You can now use [partial indexes](../v20.2/partial-indexes.html) to specify a subset of rows and columns in a table that evaluate to true on a `WHERE` filter defined at [index creation](#creation).
+Core | **`ENUM` data types** | CockroachDB now supports the creation and management of [user-defined `ENUM` data types](../v20.2/enum.html) consisting of sets of enumerated, static values.
+Core | **Materialized Views** | CockroachDB now supports [materialized views](../v20.2/views.html#materialized-views), or views that store their selection query results on-disk.
+Core | **View Replacement** | CockroachDB now supports [replacing an existing view](../v20.2/create-view.html#replace-an-existing-view) with the `CREATE OR REPLACE VIEW` syntax.  
+Core | **Foreign Key Performance and Compatibility** | When adding the [`FOREIGN KEY`](../v20.2/foriegn-key.html) constraint, it is no longer required to have an index on the referencing columns.
+Core | **`EXPLAIN` Improvements** | The response of the `EXPLAIN` statement now includes the estimated number of rows scanned by the query as well as other usability improvements.
+Core | **Disallowing Full Table Scans** | You can use the new [`disallow_full_table_scans` session variable](..v20.2/set-vars.html) to disallow full table and secondary index scans.   
+
+### Recovery and I/O
+
+Version | Feature | Description
+--------|---------|------------
+Core | **Backup/Restore in Core Version** | The core version of CockroachDB now lets you perform full cluster backups, all restore options, as well as bulk exports. Incremental backups, locality-aware backups, and other advanced backup functionality continue to require an enterprise license.
+Enterprise | **KMS Support for Encrypted Backups** | You can now use [AWS Key Management Service (KMS)](../v20.2/take-and-restore-encrypted-backups.html#use-aws-key-management-service) to encrypt the files that full or incremental backups generate.
+Enterprise | **Backup Scheduling** | You can now [create schedules for CockroachDB backups](../v20.2/manage-a-backup-schedule#create-a-new-backup-schedule), as well as [view, pause, resume, and drop backup schedules](../v20.2/manage-a-backup-schedule#view-and-control-the-backup-schedule). Once a scheduled backup is created, you can use [`SHOW BACKUP`](../v20.2/manage-a-backup-schedule#view-scheduled-backup-details) to view the backup details. Note that backup scheduling for full backups is included in the core version of CockroachDB.  
```

Comment:
Maybe we highlight this as Core and go the other way in terms of scheduling backups with Enterprise backup functionality requires a license or Cockroach Cloud?

I would suggest that we highlight `SHOW SCHEDULE` to inspect schedule status and inspect latest error if one is encountered and then say you can use `SHOW BACKUPS IN` and `SHOW BACKUP` to inspect the individual backups.

Reply:
Also, if there's prioritization here KMS is low-ish on the totem pole wrt broad reach, so I'd put it last in the IO section.